app/build.gradle

```diff
@@ -56,6 +56,10 @@ android {
         preDexLibraries = false
         javaMaxHeapSize "4g"
     }
+
+    lintOptions {
+        abortOnError false
```

Comment:
Instead of suppressing error, please try to solve it. Suppressing errors in build can result in crashes on runtime

Reply:
@iamareebjamal I tried removing without this and it fails to build. The error was due to the SDK version for parallax SDK and automatic signing, I have changed the SDK version to match that of app.

Reply:
OK